content/en/news/2019/announcing-1.4/change-notes/index.md

```diff
@@ -4,4 +4,39 @@ description: Istio 1.4 release notes.
 weight: 10
 ---
 
-TBD
+## Traffic management
+
+- **Added** support for fractional [request mirroring](/docs/tasks/traffic-management/mirroring/).
```

Comment:
I am not quite sure what you mean. https://preliminary.istio.io/docs/concepts/traffic-management/ does not have anything about mirroring. We already have a doc that explains this, and I linked to it. Its not really a "different type" of mirroring, its just sending only x% of traffic instead of 100%. Would it make sense to just rephrase this to "Added support for mirroring a percentage of traffic"?

Reply:
Yes that would be enough clarity.
The correct link for the mirroring task is this one: `https://istio.io/docs/tasks/traffic-management/mirroring/`